tests/input/t460s.pln

```diff
@@ -3,6 +3,8 @@
 local_interpreter=/bin/sh -e
 environment=WORKDIR="$PWD" http_proxy="http://192.168.3.2:3128"
 
+begin={ [ -f .lock ] && false || touch .lock; }
```

Comment:
It seems, now it's safe to remove braces here as well.

Reply:
Yes, this seems to suggest a syntax which is valid, but not nessesary